yarn-project/sequencer-client/src/config.ts

```diff
@@ -62,7 +59,6 @@ export function getConfigEnvVars(): SequencerClientConfig {
 
   return {
     rpcUrl: ETHEREUM_HOST ? ETHEREUM_HOST : '',
-    chainId: CHAIN_ID ? +CHAIN_ID : 31337, // 31337 is the default chain id for anvil
```

Comment:
Why remove the chain id from the configuration? It's usually a good practice to include the chain id as a safeguard to ensure you don't accidentally connect to the wrong network by entering an incorrect RPC URL.

Reply:
That's a good point. I added the chain Id configuration back in and validate it on Aztec Node creation.